src/www/ui_tests/api/Models/LicenseCandidateTest.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+/*
+ SPDX-FileCopyrightText: © 2024 Valens Niyonsenga <valensniyonsenga2003@gmail.com>
+
+ SPDX-License-Identifier: GPL-2.0-only
+*/
+/**
+ * @file
+ * @brief Tests for LicenseCandidateTest
+ */
+namespace Fossology\UI\Api\Test\Models;
+
+use Fossology\UI\Api\Models\LicenseCandidate;
+use Fossology\UI\Api\Models\ApiVersion;
+
+class LicenseCandidateTest extends \PHPUnit\Framework\TestCase
+{
+  /**
+   * Provides test data and an instance of the LicenseCandidate class.
+   *
+   * @return array An associative array containing test data and a LicenseCandidate object.
+   */
+  public function getLicenseCandidateInfo($version=ApiVersion::V2)
+  {
+    $id = 9;
+    $shortname = "MIT";
+    $spdxid = 2;
+    $fullname = "MIT License";
+    $text = "Permission is hereby granted, free of charge, to any person obtaining a copy...";
+    $group_name = "fossy";
+    $group_id = 3;
+
+    if($version == ApiVersion::V2){
+      $candidateInfo = [
+        'id' => $id,
+        'shortName' => $shortname,
+        'spdxId' => $spdxid,
+        'fullName' => $fullname,
+        'text' => $text,
+        'groupName' => $group_name,
+        'groupId' => $group_id,
+      ];
+    }else{
+      $candidateInfo = [
+        'id' => $id,
+        'shortname' => $shortname,
+        'spdxid' => $spdxid,
+        'fullname' => $fullname,
+        'text' => $text,
+        'group_name' => $group_name,
+        'group_id' => $group_id,
+      ];
+    }
```

Comment:
keys are wrong, please check with the `LicenseCandidate` model.

Reply:
For V1, they sould be separated by underscores.